homeassistant/components/camera/__init__.py

```diff
@@ -80,6 +87,15 @@ def async_setup(hass, config):
     hass.http.register_view(CameraMjpegStream(component.entities))
 
     yield from component.async_setup(config)
+
+    @callback
+    def update_tokens(time):
+        """Update tokens of the entities."""
+        for entity in component.entities.values():
+            entity.async_update_token()
+            entity.schedule_update_ha_state(True)
```

Comment:
That is only for stuff outside event loop. We should use `hass.async_add_job(...`